cockroachcloud/ccloud-get-started.md

```diff
@@ -0,0 +1,434 @@
+#
+# Client version: CockroachDB CCL v21.2.5 (x86_64-apple-darwin19, built 2022/02/07 21:04:05, go1.16.6)
+# Server version: CockroachDB CCL v21.2.4-1-g70835279ac (x86_64-unknown-linux-gnu, built 2022/02/03 22:31:25, go1.16.6)
+
+warning: server version older than client! proceed with caution; some features may not be available.
+
+# Cluster ID: 041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d
+#
+# Enter \? for a brief introduction.
+#
+user@free-tier7.gcp-us-central1.crdb.io:26257/defaultdb>
+~~~
+
+## Get the connection information for your cluster using `ccloud cluster sql`
+
+Use the `ccloud cluster sql` command to get connection information for the specified cluster using the cluster name.
+
+To get the [connection URL](../{{site.versions["stable"]}}/connection-parameters.html#connect-using-a-url), use the `--connection-url` option.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster sql --connection-url dim-dog
+~~~
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+~~~
+∙∙∙ Retrieving cluster info...
+postgresql://free-tier7.gcp-us-central1.crdb.io:26257/defaultdb?options=--cluster%3Ddim-dog-1567&sslmode=verify-full&sslrootcert=%2FUsers%2Fuser%2F.postgresql%2Froot.crt
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+~~~
+∙∙∙ Retrieving cluster info...
+postgresql://dim-dog-5bct.gcp-us-east4.crdb.io:26257/defaultdb?sslmode=verify-full&sslrootcert=%2FUsers%2Fuser%2FLibrary%2FCockroachCloud%2Fcerts%2Fdim-dog-ca.crt
+~~~
+
+</section>
+
+To get the individual connection parameters, use the `--connection-params` option.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster sql --connection-params dim-dog
+~~~
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+~~~
+Connection parameters
+ Database:  defaultdb
+ Host:      free-tier7.gcp-us-central1.crdb.io
+ Options:   --cluster=dim-dog-1567
+ Port:      26257
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+~~~
+∙∙∙ Retrieving cluster info...
+Connection parameters
+ Database:  defaultdb
+ Host:      dim-dog-5bct.gcp-us-east4.crdb.io
+ Port:      26257
+~~~
+
+</section>
+
+## Create a SQL user using `ccloud cluster user create`
+
+Use the `ccloud cluster user create` command to create a new SQL user by passing in the username and the cluster name.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster user create maxroach dim-dog
+~~~
+
+{% comment %}
+~~~
+Password: ****************
+∙∙∙ Creating SQL user...
+~~~
+{% endcomment %}
+
+## Delete a cluster using `ccloud cluster delete`
+
+Use the `ccloud cluster delete` command to delete the specified cluster using the cluster name.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster delete dim-dog
+~~~
+
+~~~
+∙∙∙ Deleting cluster...
+Success! Deleted cluster
+ id: 041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d
+~~~
+
+{{site.data.alerts.callout_danger}}
```

Comment:
Normally we use Warning for security issues and data loss. Maybe this could be a note instead?